apps/ade-cli/src/services/sync/syncCloudRelayStore.ts

```
@@ -88,55 +107,184 @@ export function createSyncCloudRelayStore(args: { filePath: string }) {
+      const remainingMs = deadline - Date.now();
+      if (remainingMs <= 0) return null;
+      Atomics.wait(lockWaiter, 0, 0, Math.min(IDENTITY_CONFIG_LOCK_RETRY_MS, remainingMs));
+    }
+  };
+
+  const releaseRotationLock = (lease: RotationLockLease): void => {
+    try {
+      fs.closeSync(lease.fd);
+    } finally {
+      const current = readRotationLock();
+      if (current?.owner?.token !== lease.owner.token) return;
```

Biome (2.5.3)

[error] Unsafe usage of 'return'.

(lint/correctness/noUnsafeFinally)